### DISCUSSION TITLE
multiple views on multiple monitors?

### DISCUSSION THREAD

**Discussion by pnwork** [microsoft/vscode-jupyter#11901] [Questions and Answers] [answered]
Is it possible to have multiple view of the same notebook on multiple monitors (in browser, moved tab, anywhere)? How about vs code in general - does it support that?

**Comment by DonJayamanne** [comment, marked as answer]
Unfortunately currently there's no such feature, closes i could find is https://github.com/microsoft/vscode/issues/10121
Please check issues on VS Code for mult-monitor support, the Jupyter extnsion doesn't provide such fucntionality.
